web/packages/teleport/src/Users/useUsers.ts

```diff
@@ -119,10 +121,21 @@ export default function useUsers({
     return items.map(r => r.name);
   }
 
+  function onDismissUsersMAUNotice() {
+    storageService.setUsersMAUAcknowledged();
+  }
+
   useEffect(() => {
     attemptActions.do(() => ctx.userService.fetchUsers().then(setUsers));
   }, []);
 
+  // if the cluster has billing enabled, and usageBasedBilling, and they haven't acknowledged
+  // the info yet
+  const showMAUInfo =
+    ctx.getFeatureFlags().billing &&
+    cfg.isUsageBasedBilling &&
+    !storageService.getUsersMAUAcknowledged();
+
   return {
```

Comment:
I'd like to make sure this logic is sound. I was able to spoof `isUsageBasedBilling` in my config but I'm curious to know if the `billing` feature flag should also be included here? I believe so (as only those with access to billing would want to know, yes?)

Reply:
yeah i think it makes sense to include it. you can see the conditional we use [here](https://github.com/gravitational/teleport/blob/master/lib/services/useracl.go#L173), to set the billing access

Reply:
> actually now i'm curious, can cloud clusters be NON usage based?

Yes.